pkgs/top-level/all-packages.nix

```diff
@@ -9971,6 +9971,9 @@ in
   cmake_2_8 = callPackage ../development/tools/build-managers/cmake/2.8.nix { };
 
   cmake = libsForQt5.callPackage ../development/tools/build-managers/cmake { };
+  cmake-manpages = cmake.override {
+    enableManpages = true; inherit (python3Packages) sphinx;
```

Comment:
Personally, I don't think there should be another "build" of cmake for this. Why not enable it unconditionally? The outputs being split (should) enable users to exclude this output from being installed.

Reply:
there was concern on macos see the link in my first post. I would prefer to not add a new attribute too.

Reply:
OK I see. Then perhaps we can set the "default output" to be the output `man`? This way, when one installs `cmake-manpages`, they'll truly get only man pages in their environment - this is a bit more coherent with the attribute's name IMO. Perhaps by setting either `pathsToLink` under the condition of `enableManpages`?

Reply:
Also, it'd be nice if this:

> Enabling this by default will introduce a whole bunch of python dependencies into the darwin stdenv, we had a similar problem with llvm/clang.

([Comment by @lnl7](https://github.com/NixOS/nixpkgs/pull/31344#issuecomment-342398586)) was explained in a comment somewhere.

Reply:
Take a look at clang/llvm, that has a similar problem and we handled that with a man output in a similar way as suggested here.

Reply:
Yea I think I understand the concern, it's just that one may wonder in the future what is the story behind this so I think it'd be nice to write a 2 line comment.